src/main/res/values/strings.xml

```diff
@@ -85,23 +85,23 @@
   <string name="name">Name</string>
   <string name="network_error_description">Network error.</string>
   <string name="never">Never</string>
-  <string name="new_updates_available">New updates available</string>
+  <string name="new_updates_available">New application versions available</string>
```

Comment:
Shorter strings are better for notifications. Same for `new_updates_description_format`. They're also similar to ones used in F-Droid.

I also prefer "update" than "application version", and it's not the only place where such text is used. I'd like to keep "update" word.

Reply:
Could it be "upgrade", and then looking for upgrades is "update"?

Reply:
No, it's the same process.